packages/tokens/src/index.ts

```diff
@@ -1,5 +1,7 @@
+export { Mode } from './mode';
```

Comment:
Why is this one export differently than the others?

Reply:
Because it's not exported as `default` from `./mode` (since it's both a `const` and `type`)